lib.php

```diff
@@ -6155,11 +6155,11 @@ function hsuforum_tp_add_read_record($userid, $postid) {
 /**
  * If its an old post, do nothing. If the record exists, the maintenance will clear it up later.
  *
- * @return bool
+ * @return array
```

Comment:
Still returns true in else block. Is there a reason for having mixed return type?